### ISSUE TITLE
ff_scoringhistory on Sleeper fails

### ISSUE THREAD

**Issue by msavoca90** [ffverse/ffscrapr#364]
**Describe the bug**
Unable to generate scoring history on Sleeper when attempting to scrape for use with ffsimulator

```r
conn =  sleeper_connect(season = 2022, league_id = 802734610873188352)
scoring_history = ff_scoringhistory(conn)
```
Error in `dplyr::inner_join()`:
Can't join on `x$season` x `y$season` because of incompatible types.
`x$season` is of type character.
`y$season` is of type integer.

**Comment by tanho63** [comment]
Hmm. This is because of a new issue related to nflreadr's rosters data - season is being returned as a character instead of an integer like it probably should be.

**Comment by tanho63** [comment]
Resolved with nflverse/nflfastR-roster#47

```r
R> library(ffscrapr)

R> conn =  sleeper_connect(season = 2022, league_id = 802734610873188352)

R> scoring_history = ff_scoringhistory(conn)

R> scoring_history
# A tibble: 115,540 × 35
   season  week gsis_id    sportradar_id sleeper_id player_name   pos   team  points
    <int> <int> <chr>      <chr>         <chr>      <chr>         <chr> <chr>  <dbl>
 1   1999    11 00-0016479 NA            NA         Pat Tillman   DB    ARI     0   
 2   1999     4 00-0001907 NA            NA         Dave Brown    QB    ARI     1.2 
 3   1999     5 00-0001907 NA            NA         Dave Brown    QB    ARI     1.64
 4   1999     6 00-0001907 NA            NA         Dave Brown    QB    ARI     8.28
 5   1999     8 00-0001907 NA            NA         Dave Brown    QB    ARI     2.28
 6   1999     9 00-0001907 NA            NA         Dave Brown    QB    ARI     8.26
 7   1999    10 00-0001907 NA            NA         Dave Brown    QB    ARI    10.3 
 8   1999    11 00-0001907 NA            NA         Dave Brown    QB    ARI     8.3 
 9   1999    12 00-0001907 NA            NA         Dave Brown    QB    ARI     3.44
10   1999     8 00-0006410 NA            NA         Chris Greisen QB    ARI     0.16
# … with 115,530 more rows, and 26 more variables: receiving_fumbles_lost <dbl>,
#   rushing_fumbles_lost <dbl>, sack_fumbles_lost <dbl>, carries <dbl>,
#   interceptions <dbl>, passing_2pt_conversions <dbl>, passing_tds <dbl>,
#   passing_yards <dbl>, receiving_2pt_conversions <dbl>, receiving_first_downs <dbl>,
#   receiving_tds <dbl>, receiving_yards <dbl>, receptions <dbl>,
#   rushing_2pt_conversions <dbl>, rushing_first_downs <dbl>, rushing_tds <dbl>,
#   rushing_yards <dbl>, fg_made_0_19 <dbl>, fg_made_20_29 <dbl>, fg_made_30_39 <dbl>, …
```
